.github/workflows/install.yml

```diff
@@ -11,6 +11,36 @@ defaults:
     shell: bash
 
 jobs:
+  debian:
+    runs-on: ubuntu-latest
+    container: debian
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install prerequisites
+        run: apt-get update && apt-get -y install build-essential cmake curl git
+      - name: Install Rust
+        run: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+      - name: Install Pop
+        run: |
+          . "$HOME/.cargo/env"
+          cargo install --locked --path ./crates/pop-cli
+      - name: Run Pop Install
+        run: |
+          . "$HOME/.cargo/env"
+          pop install -y
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust
+        run: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+      - name: Install Pop
+        run: |
+          . "$HOME/.cargo/env"
+          cargo install --locked --path ./crates/pop-cli
+      - name: Run Pop Install
+        run:
+          pop install -y
   ubuntu:
```

Comment:
Missing redhat in line with supported types below

Reply:
Fixed, CI: https://github.com/r0gue-io/pop-cli/actions/runs/9029075945